tests/xfstests/partition.pm

```diff
@@ -191,7 +191,15 @@ sub create_loop_device_by_rootsize {
         assert_script_run("losetup -fP $INST_DIR/$_", 300);
     }
     script_run("losetup -a");
-    format_with_options("$INST_DIR/test_dev", $para{fstype});
+    if ($para{fstype} =~ /overlay/) {
+        my $ovl_base_fs = get_var('XFSTESTS_OVL_BASE_FSTYP', 'xfs');
```

Comment:
How about change XFSTESTS_OVL_BASE_FSTYP to XFSTESTS_OVERLAY_BASE_FS?
And also add the new switch to variables.md